src/cudamatrix/cu-device.h

```diff
@@ -184,8 +184,28 @@ class CuDevice {
   /// (i.e. from outside the class), call this only if Enabled() returns true.
   bool IsComputeExclusive();
 
+  // Register command line options for CUDA device.  
+  // This must be done before calling CuDevice::Initialize()
+  // Example:
+  //  CuDevice::RegisterDeviceOptions(&po);
+  //  CuDevice::Initialize();
+  static void RegisterDeviceOptions(OptionsItf *po) {
+    CuDevice::device_options_.Register(po);  
+  }
   ~CuDevice();
  private:
+
+  struct CuDeviceOptions_t {
+    bool use_tensor_cores_; //Enable tensor cores
+    CuDeviceOptions_t () : use_tensor_cores_(false) {};
+    void Register(OptionsItf *po) {
+      po->Register("cuda-use-tensor-cores",&use_tensor_cores_, "Enable FP16 tensor math. "
```

Comment:
space after comma.
Please add: "Only recommended for inference (decoding)."
I think it would be a good idea to have these options registered in programs that might be used for inference, e.g.
nnet3-compute, if you haven't already done that.

Reply:
fixed